components/locid/src/extensions/unicode/value.rs

```diff
@@ -52,7 +52,7 @@ impl Value {
     /// Value::try_from_bytes(b"buddhist").expect("Parsing failed.");
     /// ```
     pub fn try_from_bytes(input: &[u8]) -> Result<Self, ParserError> {
-        let mut v = ShortVec::new();
+        let mut v = Vec::new();
```

Comment:
Thought: A bit of a pity that this allocates now. Not worth fixing right now.

Reply:
can we do `Vec::with_capacity(1)` here?